sklearn/multioutput.py

```diff
@@ -0,0 +1,212 @@
+"""
+MultiOneVsRestClassifier
+===========================
+This module includes several classes that extend base estimators to 
+multi-target estimators. 
+
+Most sklearn estimators use a response matrix to train a target function
+with a single output variable.  I.e. typical estimators use the 
+training set X to estimate a target function f(X) that 
+predicts a single Y.  The purpose of this class is to extend estimators
+to be able to estimate a series of target functions (f1,f2,f3...,fn)
+that are trained on a single X predictor matrix to predict a series
+of reponses (y1,y2,y3...,yn).
+"""
+
+#Author: Hugo Bowne-Anderson <hugobowne@gmail.com>
+#Author: Chris Rivera <chris.richard.rivera@gmail.com>
+#Author: Michael Williamson
+#Author: James Ashton Nichols <james.ashton.nichols@gmail.com>
+
+#License: BSD 3 clause
+
+import array
+import numpy as np
+import warnings
+import scipy.sparse as sp
+
+from sklearn.base import BaseEstimator, ClassifierMixin
+from sklearn.base import clone, is_classifier
+from sklearn.base import MetaEstimatorMixin, is_regressor
+from sklearn.preprocessing import LabelBinarizer
+from sklearn.metrics.pairwise import euclidean_distances
+from sklearn.utils import check_random_state
+from sklearn.utils.validation import _num_samples
+from sklearn.utils.validation import check_consistent_length
+from sklearn.utils.validation import check_is_fitted
+from sklearn.externals.joblib import Parallel
+from sklearn.externals.joblib import delayed
+
+
+from sklearn.multiclass import OneVsRestClassifier
+
+
+class MultiOneVsRestClassifier(BaseEstimator):
```

Comment:
this should inherit from `ClassifierMixin` so that the `isclassifier` function recognizes it as a classifier (so that while grid-searched it is a classifier, etc etc)